lib/matplotlib/tests/test_widgets.py

```diff
@@ -81,9 +82,7 @@ def onselect(epress, erelease):
     tool = widgets.RectangleSelector(ax, onselect, interactive=True,
                                      drag_from_anywhere=drag_from_anywhere)
     # Create rectangle
-    do_event(tool, 'press', xdata=0, ydata=10, button=1)
-    do_event(tool, 'onmove', xdata=100, ydata=120, button=1)
-    do_event(tool, 'release', xdata=100, ydata=120, button=1)
+    click_and_move(tool, start=[0, 10], end=[100, 120])
```

Comment:
I'd prefer tuples over lists for specifying points.

Reply:
How blocking is this comment? It's a bit of a pain to change now, but I do agree with you.

Reply:
I took the liberty to push a commit changing all the lists to tuples. It's actually not very painful at all. Regex-Replace: 1min + 1min to check the individual replacements. :)